edx_proctoring/api.py

```diff
@@ -1963,6 +1958,12 @@ def _get_proctored_exam_view(exam, context, exam_id, user_id, course_id):
             attempt['is_status_acknowledged'],
             exam
         ) else 'proctored_exam/verified.html'
+        if context.get('grace_period', False):
+            is_updated = verify_and_add_wait_deadline(context, exam['due_date'])
+            # The edge case where student has already acknowledged the result
+            # but the course team changed the grace period
+            if is_updated and not student_view_template:
+                student_view_template = 'proctored_exam/verified.html'
```

Comment:
Can we define this variable at the start of this method? 
`student_view_template = 'proctored_exam/verified.html'`

Reply:
This variable will hold either None or `verified.html` before entering into the if condition. Now, as I have mentioned that this is the edge case where the user, who has already acknowledged the status, will be able to see exam content if the course team decides to increase grace period. is_updated tells context was updated, which means wait period is there. not student_view_template means it has been acknowledged, but as we don't want the content to be seen, we override the value of template to be `verified.html`

Reply:
thank you for explaining the behavior here. This looks good as is, but I was having second thoughts if we could move the `'proctored_exam/verified.html'` in some enum variable.